Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/Results/ResultsViewController.swift

```diff
@@ -0,0 +1,60 @@
+import UIKit
+
+/// View controller for `ResultsView` screen.
+public final class ResultsViewController: UIViewController {
+    public override var nibName: String? { "ResultsView" }
+    
+    public let viewModel: ResultsViewModel
+
+    public init(viewModel: ResultsViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: "ResultsView", bundle: .module)
+    }
+    
```

Comment:
Subject to the comment above about merge order of PRs, if this comes first:
- Could this screen ever have a back button? If yes, then it should have the back button method for setting the label
- It should probably have the right bar button methods / dismiss method for setting dismiss button if it is displayed modally

If this PR is merged after the `BaseViewController` PR, then that functionality would be included for 'free' so maybe wait to see :)

Reply:
For the page I'm making this for in Wallet, the intention is that the main button (`resultsButton`) will dismiss the page. But yes, I agree that for future use we may need a back button :)